docker-compose.yml

```
@@ -0,0 +1,85 @@
+    networks:
+      - bearatlas
+    restart: unless-stopped
+
+volumes:
+  postgres_data:
+  redis_data:
+  typesense_data:
+
+networks:
+  bearatlas:
+    driver: bridge
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)